src/utils/index.ts

```diff
@@ -0,0 +1,27 @@
+/**
+ * It's important to add an appropriate role and a keyboard support
+ * for non-interactive HTML elements with click handlers, such as `<div onClick={handler}></div>`.
+ * This function returns a basic set of attributes
+ * to make the clickable element focusable and handle the Enter keypress.
+ */
+export function getClickableElementAttributes(handler: () => void) {
+    return {
+        role: 'button',
+        tabIndex: 0,
+        onClick: handler,
+        onKeyDown: (event: React.KeyboardEvent) => {
+            if (event.key === 'Enter') {
```

Comment:
For buttons space as well. And technically `keyup` is more appropriate since keydown fires repeatedly, and also click is triggered when the button is released.